fiber-opentelemetry/README.md

```diff
@@ -0,0 +1,26 @@
+### Basic Example
+
+This example shows how to trace every request (using fiber-opentelemetry package) to you fiber application.
+
+### Step by step
+
+To run this example on your local machine please follow this instruction:
+
+1. Go to https://lightstep.com, create account and follow instruction (or you can just scroll down to see screenshots)
+1. You should already have token
```

Comment:
This line and next: why do you use `1.` for all list point? Can you please make normal numerical list.

Reply:
It's easier to move points if it's always `1.` at the beginning, what's more it doesn't matter for markdown. More here: https://www.markdownguide.org/basic-syntax/#ordered-lists